internal/neofs/neofs.go

```diff
@@ -345,6 +376,52 @@ func (x *NeoFS) CreateObject(ctx context.Context, prm layer.PrmObjectCreate) (oi
 	return writer.GetResult().StoredObjectID(), nil
 }
 
+// PrepareMulptipartHeader implements neofs.NeoFS interface method.
+func (x *NeoFS) PrepareMulptipartHeader(ctx context.Context, header object.Object, metaChecksum hash.Hash, homomorphicChecksum hash.Hash, payloadLength uint64) (*object.Object, error) {
```

Comment:
according to the description to that struct, it is "connection to the NeoFS network",  do we really want it to have such helper funcs? neofs knows nothing about "multiparts"

Reply:
agree with @carpawell. This could be generic `FinalizeObjectWithPayloadChecksums` (caller can set payload size by himself)

btw typo in name

Reply:
Renamed